frontend/mr-admin-flate/src/components/filter/TiltaksgjennomforingFilterButtons.tsx

```diff
@@ -84,6 +91,16 @@ export function TiltaksgjennomforingFilterButtons({ filterAtom }: Props) {
               >
                 Opprett ny tiltaksgjennomføring
               </Lenkeknapp>
+            ) : (
+              <>
+                <Button
+                  variant="primary"
+                  size="small"
+                  onClick={() => setVisKanIkkeOppretteTiltakModal(true)}
+                >
+                  Opprett ny tiltaksgjennomføring
+                </Button>
+              </>
```

Comment:
Oppretter egen button her for å ikke måtte tacke på `handleClick` på lenken over som tar bruker til skjema for å opprette tiltak. Velger heller å være litt mer eksplisitt her.